x-pack/examples/ui_actions_enhanced_examples/public/plugin.ts

```diff
@@ -150,6 +153,12 @@ export class UiActionsEnhancedExamplesPlugin
         },
       ],
     });
+
+    const startServicesPromise = core.getStartServices();
+    registerButtonEmbeddable(
+      embeddable,
+      new Promise((resolve) => startServicesPromise.then(([_, startDeps]) => resolve(startDeps)))
```

Comment:
```suggestion
      startServicesPromise.then(([_, startDeps]) => startDeps)
```

nit: isn't it the same?

Reply:
Thanks for simplification